lua/sp.c

```diff
@@ -1116,7 +1116,7 @@ static int create_temp_table(Lua lua, pthread_mutex_t **lk, const char **name)
         return luaL_error(sp->lua, sqlite3ErrStr(SQLITE_SCHEMA));
     }
     // now, actually create the temp table
-    *lk = malloc(sizeof(pthread_mutex_t));
```

Comment:
Why calloc a mutex we're about to initialize?

Reply:
Fixed.